gigaword-indexer/src/main/java/edu/isi/vista/gigawordIndexer/IndexGigawordWithElasticSearch.java

```diff
@@ -67,6 +69,16 @@
 
   private static final String PARAM_PORT_SECONDARY = "secondaryPort";
 
+  private static final String PARAM_THRESHOLD = "threshold";
+
+  private static int total_doc = 0;
+
+  private static int index_failed = 0;
```

Comment:
Not an idiomatic variable style for Java

Reply:
Renamed variables